membership/index.md

```diff
@@ -0,0 +1,45 @@
+---
+redirect_from: 
+    - membership/founding-membership
+---
+
+# Become a member
+
+We’re currently looking for members to help shape the Foundation for Public Code’s long term future.
```

Comment:
Are the word 'currently', and perhaps even this whole sentence, still legacy from 'founding members' text? I'd replace this narrative with something less time-bound.